src/Grpc.AspNetCore.Server/Internal/ClientStreamingServerCallHandler.cs

```diff
@@ -0,0 +1,60 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using System.Threading.Tasks;
+using Grpc.Core;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Grpc.AspNetCore.Server.Internal
+{
+    internal class ClientStreamingServerCallHandler<TRequest, TResponse, TService> : ServerCallHandlerBase<TRequest, TResponse, TService>
+        where TRequest : class
+        where TResponse : class
+        where TService : class
+    {
+        public ClientStreamingServerCallHandler(Method<TRequest, TResponse> method) : base(method)
+        {
+        }
+
+        public override async Task HandleCallAsync(HttpContext httpContext)
+        {
+            httpContext.Response.ContentType = "application/grpc";
```

Comment:
Is anyone on registering this with IANA? Not in [the registry](https://www.iana.org/assignments/media-types/media-types.xml) as of today. Ref. RFC6838.

Reply:
cc @jtattermusch ?